sdk/elasticsan/Azure.ResourceManager.ElasticSan/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 1.0.0-beta.7 (Unreleased)
+## 1.0.0 (2024-01-26)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- Supported snapshot and CMK for ElasticSan
+- Supported private endpoints for ElasticSan
+- Supported ElasticSan basic operations 
 
 ### Other Changes
+This is the first stable release of ElasticSan client library.
```

Comment:
Since this is first GA, we need to verity if all SDK type names are fine.

Reply:
Anything that we should do for the verification?

Reply:
This will be done by SDK team member.